File: AirportSim/AirportSim.Domain/Dtos/AirplaneDto.cs
using AirportSim.Domain.Interfaces;
using System;

namespace AirportSim.Domain.Dtos
{
    public class AirplaneDto : IDto
    {
        public Guid Id { get; set; }
        public string Type { get; set; }
        public bool IsOutside { get; set; }
        public string Objective { get; set; }
        public string CurrentStationName { get; set; }
        public DateTimeOffset EnteredAt { get; set; }
    }
}

File: AirportSim.Client/ClientApp/src/app/dtos/airplane-dto.ts
export class Guid{
    static _empty = "00000000-0000-0000-0000-000000000000";
    public static get empty():string{
        return this._empty;
    }
}

export interface AirplaneDto {
    id:string,
    type:string,
    isOutside:boolean,
    objective:string,
    currentStationName?:string,
    enteredAt:Date,
    color:string
}
